docs/porting/third-party-deps.md

```diff
@@ -0,0 +1,52 @@
+# Analyzing your 3rd Party Dependencies
+
+By [Phillip Carter](https://github.com/cartermp)
+
+The first step int he porting process is to understand your third party dependencies, figure out which of them (if any), don't run on .NET Core, and developing a contingency plan for those which don't run on .NET Core.
+
+## Prerequisites
+
+This article will assume you are using Windows and Visual Studio  If you depend on Nuget packages, you may also wish to download the [Nuget Package Explorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer).
```

Comment:
This needs some revision; not only is NuGet covered next, but the "if you depend..." makes it seem like an edge case that won't be covered here. Better to say something like, "Your dependencies may be NuGet packages or assemblies in the .NET Framework Class Library or the file system."